server/render/index.js

```diff
@@ -86,8 +86,7 @@ export function serverRender( req, res ) {
 		config.isEnabled( 'server-side-rendering' ) &&
 		context.layout &&
 		! context.user &&
-		isEmpty( context.query ) &&
```

Comment:
Again, dropping this criterion might have quite an impact since we might end up calling `renderToString` (and thus, blocking `node`) way more often.

cc @ehg